packages/accordion/src/Default/Default.types.ts

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import * as bs from 'react-bootstrap';
+
+export interface AccordionItemProps {
+  id: string;
+  label: string;
+  view: React.ReactNode;
+}
+
+export interface AccordionDefaultCommons {
+  theme?: 'Default' | 'Dark';
+  size?: 'Sm' | 'Md' | 'Lg';
+  prefix?: string;
+  items: AccordionItemProps[];
+}
+
+export interface themePrefixesProps {
+  [key: string]: string;
+}
+
+export type AccordionDefaultProps = AccordionDefaultCommons &
```

Comment:
Remember to intersect the `AccordionProps` here and remove the `React.AllHTMLAttributes<HTMLDivElement>`, that's too generic for this component.

```ts
export type AccordionDefaultProps = AccordionDefaultCommons & bs.AccordionProps;
```